bot/exts/fun/quack.py

```diff
@@ -0,0 +1,80 @@
+import random
+from typing import Literal, Optional
+
+import discord
+from aiohttp import client_exceptions
```

Comment:
btw, I think you're importing a file here? `ClientConnectorError` is most likely revealed under the `aiohttp` namespace meaning `aiohttp.ClientConnectorError` should be enough.

That said this should be remove per our previous comments